rskj-core/src/main/java/org/ethereum/vm/program/ProgramResult.java

```diff
@@ -224,6 +249,22 @@ public void rejectLogInfos() {
         }
     }
 
+    // This is the actual refunded amount of gas.
+    // It should never be higher than half of the amount of gas consumed.
+    public void addDeductedRefund(long gasValue) {
+        deductedRefund += gasValue;
```

Comment:
Again, should use `GasCost.add` to avoid overflows.

Reply:
https://github.com/rsksmart/rskj/pull/1548/commits/893b89fe9e6096b815bae692aa5b35ab05f03dce